repository/databases/runtime.go

```diff
@@ -0,0 +1,51 @@
+package databases
+
+import (
+	"context"
+
+	"github.com/blcvn/corev4-explorer/appconfig"
+	connector "github.com/blcvn/corev4-explorer/connectors/database"
+	"github.com/blcvn/corev4-explorer/repository/databases/oracle"
+)
+
+func NewDataDBRepository(ctx context.Context) (dataDB, error) {
+	switch appconfig.OnchainDbConfig["type"] {
+	case "ORACLE":
+		client, err := connector.InitOracleClient(appconfig.OnchainDbConfig)
```

Comment:
Refactor the `NewDataDBRepository` and `NewTaskDBRepository` functions to avoid code duplication in the `default` case. Extract the common logic into a separate function if the `default` case should behave the same as the `ORACLE` case. [medium]